.github/workflows/code_checks.yaml

```diff
@@ -11,36 +11,68 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                php: ['7.2', '7.3', '7.4']
+                php:
+                    - "7.2"
+                    - "7.3"
+                    - "7.4"
+                symfony:
+                    - 2.*
```

Comment:
Not sure there is a point to support symfony below v4, as min php versions are different. While we can leave as is, for a while, eventually it will be dropped..
What do U think?

Reply:
I guess you are right that the probability of someone using PHP >= 7.2 is still stuck at Symfony 2.8 is rather low, but since we still include ~2.8 in composer, I guess we also should test for it. But yeah, for a next major version, we should definitely drop unsupported Symfony versions (reduce to 3.4|4.*|5.* or similar).